src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -1062,6 +1065,7 @@ export default class DefaultAudioVideoController
     const groupIdsToReceive = context.videosToReceive.array().map((streamId: number) => {
       return context.videoStreamIndex.groupIdForStreamId(streamId);
     });
+    this.receiveRemotePauseResumeTask.updateSubscribedGroupdIds(new Set(groupIdsToReceive));
```

Comment:
1. Are there any existing observers that `receiveRemotePauseResumeTask` itself can implement and on update, call the `updateSubscribedGroupdIds`? Will help avoid direct dependency on DAVController.
2. When cleaning up probably as part of retry as well as under the [destroy](https://github.com/aws/amazon-chime-sdk-js/blob/bf22ce0e55b69348391144c6f739e12c10933970/src/audiovideocontroller/DefaultAudioVideoController.ts#L183) method in here, should we set the `receiveRemotePauseResumeTask` to `undefined`?

Reply:
1. This set needs to be synchronized with the state in DAVC so setting it elsewhere just leads to risk of race conditions, especially if we refactor to reuse transceivers. The existence of injectable downlink policies is why we originally had to shove this logic in DAVC but it would probably be refactored out when/if we remove that pattern.
2. It would survive since it would be reset on the first subscribe, but it doesn't hurt to be more explicit, good point. This honestly makes me wonder if we should have a checklist/gotcha list in PR template around surviving reconnects lol.

Reply:
I did not bother setting to `undefined`, but I reset it on reconnect. TBH i think the logic we have is out of sync between reconnect and initial connect...not clear why that path wasn't shared.

Reply:
I see, probably a v4 tracking point to sync it 😅 .